docs/toc.yml

```
@@ -637,6 +637,9 @@ items:
   - name: Outlook add-in tutorial
     href: tutorials/outlook-tutorial.md
     displayName: Outlook
+  - name: Develop Outlook add-ins for the new Outlook on Windows (preview)
+    href: outlook/one-outlook.md
+    displayName: Outlook
```

Per [docs guidance](https://review.learn.microsoft.com/en-us/help/contribute/preview-content?branch=main#headings-titles-and-tocs), exclude "preview" tag from TOC name. However, you can add "preview" in displayName so users' TOC search for preview content will still list this entry.
```suggestion
  - name: Develop Outlook add-ins for the new Outlook on Windows
    href: outlook/one-outlook.md
    displayName: Outlook, preview
```